### PR TITLE
ch1115 OLED display driver

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Samples/Ch1115_Sample/MeadowApp.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            graphics.CurrentFont = new Font8x8();
+
+            return base.Initialize();
+        }
+
+        public override Task Run()
+        {
+            graphics.Clear();
+            graphics.DrawTriangle(10, 10, 50, 50, 10, 50, Meadow.Foundation.Color.Red);
+            graphics.DrawRectangle(20, 15, 40, 20, Meadow.Foundation.Color.Yellow, true);
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Samples/Ch1115_Sample/MeadowApp.cs:39]: if its Monochrome, especifying colors are not needed, right?

File: Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Driver/Ch1115.Enums.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            OscFrequencyDataSet = 0xA0,
+
+            ComLevelModeSet = 0xDB,
+            ComLevelDataSet = 0x40,
+
+            DCModeSet = 0xAD,
+            DCOnOffSet = 0x8B,
+
+
+
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Driver/Ch1115.Enums.cs:87]: Empty lines here

**Review by jorgedevs — APPROVED** [review]

File: Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Samples/Ch1115_Sample/MeadowApp.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Samples/Ch1115_Sample/MeadowApp.cs:39]: uuhh ... correct ... I'll fix ;)

File: Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Driver/Ch1115.Enums.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Driver/Ch1115.Enums.cs:87]: on it

**Pull request merged**